src/deploy/db/mongo-adapter/constants.ts

```diff
@@ -11,5 +11,6 @@ export const VERSION_TYPES = {
 };
 
 // TODO dep: move these to default ENV config
-export const mongoURILocal = "mongodb://localhost:27018";
-export const mongoDbName = "zns-campaign";
+export const DEFAULT_MONGO_URI = "mongodb://localhost:27018";
+export const DEFAULT_MONGO_DB_NAME = "zns-campaign";
+export const DEFAULT_MONGO_VERSION = "7.0";
```

Comment:
remove this. in test if the version is not specified, it will create it here:
https://github.com/zer0-os/zNS/blob/feat/deploy-campaign/src/deploy/db/mongo-adapter/mongo-adapter.ts#L129-L132